internal/services/v1/permissions_test.go

```diff
@@ -1273,3 +1273,164 @@ func randString(length int) string {
 	}
 	return string(b)
 }
+
+func TestCheckPermissionWithCaveatedDebugInfoHasPermission(t *testing.T) {
```

Comment:
I'd suggest moving this instead into a new `debug_test.go`

Reply:
Why? its part of the V1 API

Reply:
yeah, I get it, but it's exercising code that lives in `v1/debug.go`